README.md

```diff
@@ -104,70 +104,128 @@ that more automatic in the end.
 ```
-python runner.py --workflow ${workflow} --json metadata/test_w_dj.json 
+python runner.py --workflow ${workflow} --json metadata/test_w_dj.json --campaign Rereco17_94X --year 2017
 ```
 
-### b-SFs 
+More options for `runner.py` 
+<details><summary>more options</summary>
+<p>
+
+```
+--wf {validation,ttcom,ttdilep_sf,ttsemilep_sf,emctag_ttdilep_sf,ctag_ttdilep_sf,ectag_ttdilep_sf,ctag_ttsemilep_sf,ectag_ttsemilep_sf,ctag_Wc_sf,ectag_Wc_sf,ctag_DY_sf,ectag_DY_sf}, --workflow {validation,ttcom,ttdilep_sf,ttsemilep_sf,emctag_ttdilep_sf,ctag_ttdilep_sf,ectag_ttdilep_sf,ctag_ttsemilep_sf,ectag_ttsemilep_sf,ctag_Wc_sf,ectag_Wc_sf,ctag_DY_sf,ectag_DY_sf}
+                        Which processor to run
+  -o OUTPUT, --output OUTPUT
+                        Output histogram filename (default: hists.coffea)
+  --samples SAMPLEJSON, --json SAMPLEJSON
+                        JSON file containing dataset and file locations
+                        (default: dummy_samples.json)
+  --year YEAR           Year
+  --campaign CAMPAIGN   Dataset campaign, change the corresponding correction
+                        files
+  --isCorr              Run with SFs
+  --isJERC              JER/JEC implemented to jet
+  --executor {iterative,futures,parsl/slurm,parsl/condor,parsl/condor/naf_lite,dask/condor,dask/slurm,dask/lpc,dask/lxplus,dask/casa}
+                        The type of executor to use (default: futures). 
+  -j WORKERS, --workers WORKERS
+                        Number of workers (cores/threads) to use for multi- worker executors (e.g. futures or condor) (default:
+                        12)
+  -s SCALEOUT, --scaleout SCALEOUT
+                        Number of nodes to scale out to if using slurm/condor.
+                        Total number of concurrent threads is ``workers x
+                        scaleout`` (default: 6)
+  --memory MEMORY       Memory used in jobs  ``(default: 4GB)
+  --disk DISK           Disk used in jobs  ``(default: 4GB)
+  --voms VOMS           Path to voms proxy, made accessible to worker nodes.
+                        By default a copy will be made to $HOME.
+  --chunk N             Number of events per process chunk
+  --retries N           Number of retries for coffea processor
+  --validate            Do not process, just check all files are accessible
+  --skipbadfiles        Skip bad files.
+  --only ONLY           Only process specific dataset or file
+  --limit N             Limit to the first N files of each dataset in sample
+                        JSON
+  --max N               Max number of chunks to run in total
+```
+</p>
+</details>
+
+Roadmap for running the tool
+
+1. Is the `.json` files ready? If not create it through [Make the json files](#make-the-json-files) section with naming scheme
+
+2. If you want JERC and SFs implemented, edit the correction config under `BTVNanoCommissioning/src/utils/AK4_parameters.py`, add a new `dict` contains correction files in dedicated campaign.
```

Comment:
Suggest to rewrite:
```
If you want to add JEC and JER to a new campaign, add a new `dict` entry in `BTVNanoCommissioning/src/utils/AK4_parameters.py` and add the JERC correction file (format: `.pkl.gz`) as entry `JME`.
```

Reply:
SFs, pu weight are also included in `AK4_parameters.py`. I correct the sentences as following

> If you want to add JEC, JER, and other correction files(SFs, pileup weight) to a new campaign, add a new `dict` entry in `BTVNanoCommissioning/src/utils/AK4_parameters.py` and add the JERC correction file (format: `.pkl.gz`) as entry `JME`.

Reply:
Sounds good to me :+1: minor remarks: 

- there is a space missing before `(SFs` 
- the last part of the sentence only refers to JERC and needs to be updated then as well

Reply:
Hmmm.. indeed `AK4_parameters.py` also contains lumimasks. Let me rewrite the whole sentences, and add a dedicated chapter for config.

2. Put the `lumiMask`, correction files (SFs, pileup weight), and JEC, JEC files under the `dict` entry in `BTVNanoCommissioning/src/utils/AK4_parameters.py`. See the details in --> point to config chapter

Reply:
:+1:  pplease change `JEC, JEC files` to `JEC, JER files`